test/integration/stats_test.rb

```diff
@@ -2,7 +2,8 @@
 
 class StatsTest < SystemTest
   test "page params is not integer" do
-    visit '/stats?page="3\">XSS"'
+    path_with_xss = '/stats?page="3\">XSS"'
+    visit URI.encode(path_with_xss)
```

Comment:
I think that by escaping the characters, this test no longer checks for XSS? It might be that Rails now has better XSS protection built in, or it might be that we need to run this test at a lower level than the `visit` API.